specification/edge/resource-manager/Microsoft.Edge/sites/readme.md

```diff
@@ -0,0 +1,91 @@
+# edgesites
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for edge sites.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+## Suppression
+
+``` yaml
+directive:
+  - suppress: MISSING_APIS_IN_DEFAULT_TAG
```

Comment:
> MISSING_APIS_IN_DEFAULT_TAG [](http://example.com/codeflow?start=13&length=28)

Specifying this is required for some important automation to run whichis required for signoff.

Reply:
Removed MISSING_APIS_IN_DEFAULT_TAG suppression from readme